pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -876,6 +876,454 @@ func TestFilterPlugins(t *testing.T) {
+					name: "TestPlugin",
+					inj:  injectedResult{PreBindStatus: int(Error)},
+				},
+			},
+			wantStatus: NewStatus(Error, `error while running "TestPlugin" prebind plugin for pod "": injected status`),
+		},
+		{
+			name: "UnschedulablePreBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj:  injectedResult{PreBindStatus: int(UnschedulableAndUnresolvable)},
+				},
+			},
+			wantStatus: NewStatus(Error, `error while running "TestPlugin" prebind plugin for pod "": injected status`),
+		},
+		{
+			name: "SuccessErrorPreBindPlugins",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj:  injectedResult{PreBindStatus: int(Success)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj:  injectedResult{PreBindStatus: int(Error)},
+				},
+			},
+			wantStatus: NewStatus(Error, `error while running "TestPlugin 1" prebind plugin for pod "": injected status`),
+		},
+		{
+			name: "ErrorSuccessPreBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj:  injectedResult{PreBindStatus: int(Error)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj:  injectedResult{PreBindStatus: int(Success)},
+				},
+			},
+			wantStatus: NewStatus(Error, `error while running "TestPlugin" prebind plugin for pod "": injected status`),
+		},
+		{
+			name: "SuccessSuccessPreBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj:  injectedResult{PreBindStatus: int(Success)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj:  injectedResult{PreBindStatus: int(Success)},
+				},
+			},
+			wantStatus: nil,
+		},
+		{
+			name: "ErrorAndErrorPlugins",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj:  injectedResult{PreBindStatus: int(Error)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj:  injectedResult{PreBindStatus: int(Error)},
+				},
+			},
+			wantStatus: NewStatus(Error, `error while running "TestPlugin" prebind plugin for pod "": injected status`),
+		},
+		{
+			name: "UnschedulableAndSuccessPreBindPlugin",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj:  injectedResult{PreBindStatus: int(Unschedulable)},
+				},
+				{
+					name: "TestPlugin 1",
+					inj:  injectedResult{PreBindStatus: int(Success)},
+				},
+			},
+			wantStatus: NewStatus(Error, `error while running "TestPlugin" prebind plugin for pod "": injected status`),
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			registry := Registry{}
+			configPlugins := &config.Plugins{PreBind: &config.PluginSet{}}
+
+			for _, pl := range tt.plugins {
+				tmpPl := pl
+				if err := registry.Register(pl.name, func(_ *runtime.Unknown, _ FrameworkHandle) (Plugin, error) {
+					return tmpPl, nil
+				}); err != nil {
+					t.Fatalf("Unable to register pre bind plugins: %s", pl.name)
+				}
+
+				configPlugins.PreBind.Enabled = append(
+					configPlugins.PreBind.Enabled,
+					config.Plugin{Name: pl.name})
```

Comment:
Can you check this style of code in this file, and make it to:

```go
xyz = append(
    configPlugins.xyz,
    config.Plugin{},
)
```

Reply:
I am not sure what you mean. Please, can you clarify further?

Reply:
I mean the ending `)` is preferred to be put in a new line:

```go
xyz = append(
    configPlugins.xyz,
    config.Plugin{})
```

=>

```go
xyz = append(
    configPlugins.xyz,
    config.Plugin{},
)
```